### PR TITLE
JavaDoc test

### PR DESCRIPTION
**Pull request by BeeHiveJava**

### CODE REVIEW

File: api/src/main/java/nl/pts44/api/User.java [hunk 1]
```diff
@@ -15,6 +15,10 @@ public User(int id, String username) {
         this.username = Objects.requireNonNull(username);
     }
 
+    public void allTheThings() {
```
Comment (TehMessiah) [inline api/src/main/java/nl/pts44/api/User.java]: BLOCKER Document this public method. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AUndocumentedApi)
CRITICAL Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1186)

File: core/src/main/java/nl/pts44/core/UserDeleter.java [hunk 2]
```diff
@@ -13,4 +13,8 @@ public void delete(User user) {
         Objects.requireNonNull(user);
     }
 
+    public void allTheThings() {
```
Comment (TehMessiah) [inline core/src/main/java/nl/pts44/core/UserDeleter.java]: CRITICAL Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1186)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 3 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 2 critical

Watch the comments in this conversation to review them.

[BLOCKER]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL'

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

File: api/src/main/java/nl/pts44/api/User.java (hunk 1 above)
Comment (TehMessiah) [inline api/src/main/java/nl/pts44/api/User.java]: BLOCKER Document this public method. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AUndocumentedApi)
CRITICAL Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1186)

File: core/src/main/java/nl/pts44/core/UserDeleter.java (hunk 2 above)
Comment (TehMessiah) [inline core/src/main/java/nl/pts44/core/UserDeleter.java]: CRITICAL Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1186)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 3 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 2 critical

Watch the comments in this conversation to review them.

[BLOCKER]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL'

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by TehMessiah** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 2 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL'

**Comment by BeeHiveJava** [comment]
retest this please

**Comment by TehMessiah** [comment]
SonarQube analysis reported 5 issues
* ![BLOCKER][BLOCKER] 3 blocker
* ![CRITICAL][CRITICAL] 2 critical

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [UserDeleter.java#L12](https://github.com/PTS4-S44/test/blob/c6fe9ec6ed35e387a78e0d38390a609f70f3c4d0/core/src/main/java/nl/pts44/core/UserDeleter.java#L12): Document this public method. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

[BLOCKER]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**